contrib/patterns/_grid-media.scss

```diff
@@ -0,0 +1,16 @@
+$medium-screen: 1000px;
```

Comment:
I think Hound is talking about Line 12…unsure why the comment was placed up here.